enhancements/node-tuning/containerize-tuned.md

```diff
@@ -0,0 +1,260 @@
+  Before=kubelet-dependencies.target
+  ConditionPathExists=/var/lib/ocp-tuned/image.env
+  
+  [Service]
+  Type=exec
+  Restart=on-failure
+  RestartSec=10
+  ExecReload=/bin/pkill --signal HUP --pidfile /run/tuned/tuned.pid
+  #TimeoutStartSec=5
+  ExecStartPre=/bin/bash -c " \
+    mkdir -p /run/tuned "
+  ExecStart=/usr/bin/podman run \
+      --rm \
+      --name openshift-tuned \
+      --privileged \
+      --authfile /var/lib/kubelet/config.json \
+      --net=host \
+      --pid=host \
+      --security-opt label=disable \
+      --volume(s)
+	… 
+      --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","-v=1"]' \
+      $NTO_IMAGE
+  Environment=PODMAN_SYSTEMD_UNIT=%n
+  EnvironmentFile=-/var/lib/ocp-tuned/image.env
+  ExecStop=/usr/bin/podman stop -t 20 openshift-tuned
+  ExecStopPost=/usr/bin/podman rm -f --ignore openshift-tuned
+
+```
+
+This approach would work even for deployments such as HyperShift, openshift-ansible and likely even IBM ROKS; this still wouldn't work for custom deployments with RHEL.
+
+* Podmanized TuneD will be started via a systemd.path based on a presence of an environment file containing the NTO release payload image (once NTO operand extracts it and writes it to disk).
+
+* Prior to the tuned main process, all necessary tuned directories and known profiles will be created and placed on the node.
+
+* This containerized approach requires having a path to custom tuned profiles directory configurable or at least in subdirectory /etc/tuned/profiles, in order to mount them as a whole in containerized environments. 
+This will have to be implemented on [Tuned](https://github.com/redhat-performance/tuned) - [RHEL-26157](https://issues.redhat.com/browse/RHEL-26157)
+
+
+### Workflow Description
+
+N/A
+
+### API Extensions
+
+No API changes
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+TBD
+
+#### Standalone Clusters
+
+#### Single-node Deployments or MicroShift
+
+How does this proposal affect the resource consumption of a
+single-node OpenShift deployment (SNO), CPU and memory?
+
+TBD
+
+### Implementation Details/Notes/Constraints
+
+* “Accumulating” NTO images during upgrades: by using podman to run TuneD from the NTO image that comes from the release payload,this old payload image will not be removed from the cluster during upgrades as it is used by the systemd unit running podman.
+* Even with “podmanized” TuneD, we’ll likely not be able to resolve issues such as [these](https://github.com/openshift/machine-config-operator/pull/2944#issuecomment-1029739561) and centralized the tuning into NTO – chicken-and-egg with default openshift TuneD profiles. Rendering them via MCO/installer, doing some built-in NTO bootstrap or using host-native TuneD would help.
+* Even with this approach, TuneD still must finish tuning before Kubelet starts or there is a chance the workloads will start to a partially configured system state.
```

Comment:
This is a requirement that this service finishes before launching any workloads.